Python 3.8 warns about incorrect use of "is not" with a literal.

--- a/jupyter_client/manager.py	2019-09-13 16:11:50.000000000 -0600
+++ b/jupyter_client/manager.py	2019-09-26 09:07:57.295032661 -0600
@@ -78,7 +78,7 @@ class KernelManager(ConnectionFileMixin)
 
     @property
     def kernel_spec(self):
-        if self._kernel_spec is None and self.kernel_name is not '':
+        if self._kernel_spec is None and self.kernel_name != '':
             self._kernel_spec = self.kernel_spec_manager.get_kernel_spec(self.kernel_name)
         return self._kernel_spec
 
